src/calendar.js

```diff
@@ -198,6 +198,29 @@ class ICalCalendar {
     }
 
 
+    /**
+     * Set/Get your feed's calscale
+     *
+     * @param {string} [scale] CALSCALE
+     * @example cal.calscale('gregorian');
+     * @since 1.7.3
+     * @returns {ICalCalendar|String}
+     */
+    calscale(scale) {
```

Comment:
I don't like `calscale` as a method name.  The method is attached to the calendar object, so I would simply prefer `scale`.